modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/NavigationBar.js

```diff
@@ -22,21 +23,34 @@ export default function NavigationBar({
 			inverted={inverted}
 			triggerLabel={navigationItems.find(({active}) => active)?.label}
 		>
-			{navigationItems.map(({active, href, label}, index) => {
-				return (
-					<ClayNavigationBar.Item
-						active={active}
-						data-nav-item-index={index}
-						key={label}
-					>
-						{href ? (
-							<ClayLink href={href}>{label}</ClayLink>
-						) : (
-							<ClayButton>{label}</ClayButton>
-						)}
-					</ClayNavigationBar.Item>
-				);
-			})}
+			{navigationItems.map(
+				({active, deprecated = false, href, label}, index) => {
+					const LinkOrButton = href ? ClayLink : ClayButton;
```

Comment:
If this was TypeScript, I think it would complain, as `ClayButton` cannot have `href` prop on L36, even if value is `undefined`. Can we use `LinkOrButton` from Clay, like we do [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/Table.tsx#L7)?

Reply:
Sure! I just updated it